brave/brave-core
# Added Microsoft IE

Resolves https://github.com/brave/brave-browser/issues/27557

<img width="701" alt="image" src="https://user-images.githubusercontent.com/2965009/214074515-52a6ff77-3139-4861-bef6-d9744489d9ff.png">

## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- go to brave://welcome page when Microsoft Internet explorer installed, windows
- check it has correct icon and name
